lib/JpegTran.js

```diff
@@ -1,33 +1,40 @@
 var childProcess = require('child_process'),
     Stream = require('stream').Stream,
     util = require('util'),
-    which = require('which'),
-    memoizeAsync = require('memoizeasync');
+    which = require('which');
 
-function JpegTran(jpegTranArgs) {
+function JpegTran(jpegTranArgs, jpegTranBinary) {
     Stream.call(this);
 
     this.jpegTranArgs = jpegTranArgs;
 
     this.writable = this.readable = true;
 
     this.hasEnded = false;
+
+    this.jpegBinary = jpegTranBinary;
 }
 
 util.inherits(JpegTran, Stream);
 
-JpegTran.getBinaryPath = memoizeAsync(function (cb) {
```

Comment:
`JpegTran.getBinaryPath` (the static method) needs to stay, it's being used here: https://github.com/assetgraph/assetgraph-builder/blob/master/lib/transforms/processImages.js#L25